spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java

```diff
@@ -545,6 +564,11 @@ else if (!Modifier.isPublic(method1.getModifiers())) {
 				}
 				return;
 			}
+			if (AnnotationUtils.getAnnotation(method1, Default.class) != null) {
```

Comment:
Should we use `findAnnotation()` here, or do you think one level of meta-annotation is enough for this purpose?

I am leaning that way - having `@Default` on some obscure meta-meta-meta... could be confusing.